solaris/solaris11/postinstall.sh

```diff
@@ -3,10 +3,28 @@
 # Wazuh, Inc 2015-2020
 
 if [ -d <INSTALL_PATH>/logs/ossec ]; then
-  mv <INSTALL_PATH>/logs/ossec/* <INSTALL_PATH>/logs/wazuh
+  mv <INSTALL_PATH>/logs/ossec/* <INSTALL_PATH>/logs/wazuh > /dev/null 2>&1 || true
```

Comment:
Why?

Reply:
If exists `logs/ossec` but is empty this command returns an error message